ProblemsSolution/ProblemsSolution.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace ProblemsSolution
+{
+    public class ArrayProblemsSolution
+    {
+        public static int SumArrayEvenNumbers(int[] numbers)
+        {
+            int sum = 0;
+            foreach(int number in numbers)
+            {
+                if(number % 2 == 0) sum += number;
+            }
+
+            return sum;
+        }
+
+        public static int[] BigestTwoNumbersArray(int[] numbers)
+        {
+            if(numbers.Length == 0) return new int[] {};
+            if(numbers.Length == 1) return new int[] {numbers[0]};
+
+            int secondBiggest = Int32.MinValue;
+            int biggest = Int32.MinValue;
+
+            foreach(int number in numbers)
+            {
+                if(number > biggest)
+                {
+                    secondBiggest = biggest;
+                    biggest = number;
+                } else if(number > secondBiggest)
+                {
+                    secondBiggest = number;
+                }
+            }
+
+            return new int[] {secondBiggest, biggest};
+        }
+
+        public static bool DuplicatesInArray(int[] numbers)
+        {
+            int indexSum = 0;
+            int arraySum = 0;
+
+            for(int i=1; i <= numbers.Length; i++){
+                indexSum += i;
+                arraySum += numbers[i - 1];
+            }
+
+            if(indexSum == arraySum)
+            {
+                return false;

```

Comment:
The ternary operator could cut up a few lines of code here.

Reply:
> The ternary operator could cut up a few lines of code here.

Fixed fb48401177edc8163c9370bff510a8dc81099111.